lib/class-wp-rest-widget-areas-controller.php

```diff
@@ -84,6 +84,53 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Retrieves the comment's schema, conforming to JSON Schema.
+	 *
+	 * @since 4.7.0
+	 *
+	 * @return array
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => 'Widget Area',
+			'type'       => 'object',
+			'properties' => array(
+				'content'           => array(
+					'description' => __( 'The content for the object.' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'arg_options' => array(
+						'sanitize_callback' => null,
+						'validate_callback' => null,
+					),
+					'properties'  => array(
+						'raw'           => array(
+							'description' => __( 'Content for the object, as it exists in the database.' ),
+							'type'        => 'string',
+							'context'     => array( 'view', 'edit', 'embed' ),
+						),
+						'rendered'      => array(
+							'description' => __( 'HTML content for the object, transformed for display.' ),
+							'type'        => 'string',
+							'context'     => array( 'view', 'edit', 'embed' ),
+							'readonly'    => true,
+						),
+						'block_version' => array(
+							'description' => __( 'the current version of the block format that the content string is usin' ),
```

Comment:
```suggestion
							'description' => __( 'Version of the content block format used by the object.' ),
```

I think this should probably match the existing posts controller.